EntAppProject/src/main/resources/templates/JBIntroSurvey.html

```diff
@@ -2,38 +2,28 @@
 
 <html>
 
-<head>
-
-<title>JobBook - Introduction Survey</title>
-
-<link rel="stylesheet" href="JobBookCSS.css">
-
-</head>
-
-<body>
- 
-
-<h1><span>Job</span>Book</h1>
-<h2>Welcome to JobBook</h2>
- 
-<p>
-In order to best assist you with finding jobs, please check the boxes that best
-describe your skillset(s).  You'll be able to clarify your skills in more detail
-on the edit profile page.
-</p>
-
-<form action="/JBUserProfile">
- 
-<input type= "checkbox" name="homeMaintenance" value="HomeMaintenance">Home Maintenance<br>
-<input type= "checkbox" name="lawn&Garden" value="Lawn&Garden">Lawn and Garden<br>
-<input type= "checkbox" name="electronics" value="Electronics">Electronics<br>
-<input type= "checkbox" name="child&PetCare" value="Child&PetCare">Child and Pet Care<br>
-<input type= "checkbox" name="automobileMechanics" value="AutomobileMechanics">Automobile Mechanics<br>
- 
-<button type="submit">Submit</button>
-
-</form>
-  
-</body>
-
-</html>
```

Comment:
I redid the HTML page restructuring from before but with the change of using the `page-title` class on span elements to specify that it should use the style included in `JobBookCSS.css`.